src/css/math.less

```diff
@@ -241,14 +241,45 @@
   }
 
   .mq-numerator {
-    padding: 0 0.1em;
+    padding: .1em .1em 0 .1em;
+    width: 100%;
+    margin-bottom: 0.2em;
   }
 
   .mq-denominator {
-    border-top: 1px solid;
-    float: right; // take out of normal flow to manipulate baseline
+    float: left;
     width: 100%;
-    padding: 0.1em;
+    padding: .15em .1em 0 .1em;
+    margin-right: -.1em;
+    margin-left: -.1em;
+    text-align: center;
+  }
+
+  .mq-denominator {
+    border-top: 1px solid;
+  }
+
+  ////
+  // \overset & \underset
+  // overset and underset text
+  .mq-overunder {
+    text-align: center;
+    position: relative;
+    top: .01em;
+
+    // fix \mathbf vertical alignment issue
+    .mq-font {
+      font-family: inherit;
```

Comment:
this causes Symbola to be used, right? There's no proper bold and italics and other styling for Symbola; what was the vertical alignment issue? Is it also fixed by fixing https://github.com/mathquill/mathquill/pull/556/files#r55783521 ?